sections/Claim/ConvertSection/ConvertSection.tsx

```diff
@@ -0,0 +1,296 @@
+import Link from 'next/link';
+import { ethers } from 'ethers';
+import { wei } from '@synthetixio/wei';
+import styled from 'styled-components';
+import { useState, useEffect, useCallback, useMemo } from 'react';
+
+import Connector from 'containers/Connector';
+import TransactionData from 'containers/TransactionData';
+import TransactionNotifier from 'containers/TransactionNotifier';
+
+import VAelinTokenContract from 'containers/ContractsInterface/contracts/vAelinToken';
+import VAelinConverterContract from 'containers/ContractsInterface/contracts/VAelinConverter';
+
+import Button from 'components/Button';
+import { FlexDivColCentered } from 'components/common';
+import ConfirmTransactionModal from 'components/ConfirmTransactionModal';
+
+import ROUTES from 'constants/routes';
+import { GasLimitEstimate } from 'constants/networks';
+import { DEFAULT_NETWORK_ID } from 'constants/defaults';
+import { TransactionStatus } from 'constants/transactions';
+
+import { getKeyValue } from 'utils/helpers';
+import { getGasEstimateWithBuffer } from 'utils/network';
+
+const ConvertSection = () => {
+	const [hasAllowance, setHasAllowance] = useState<boolean>(false);
+	const [hasVAelinBalance, setHasVAelinBalance] = useState<boolean>(false);
+	const [showConverterTxModal, setConverterTxModal] = useState<boolean>(false);
+	const [gasvAelinConverterLimitEstimate, setvAelinConverterGasLimitEstimate] =
+		useState<GasLimitEstimate>(null);
+
+	const { gasPrice, setGasPrice, setTxState } = TransactionData.useContainer();
+	const { monitorTransaction } = TransactionNotifier.useContainer();
+	const { walletAddress, network, signer } = Connector.useContainer();
+
+	const vAelinConverterContract = useMemo(() => {
+		if (!signer || !network?.id) return null;
+
+		const vAelinConverterContract = (getKeyValue(VAelinConverterContract) as any)(
```

Comment:
can you please use `ContractsInterface` for this contract initialisation? I mentioned it the other day.

Reply:
Done